src/pages/Dashboard/Markets/BorrowMarket/index.tsx

```diff
@@ -23,21 +25,47 @@ export const BorrowMarketUi: React.FC<IBorrowMarketUiProps> = ({
   isXvsEnabled,
   userTotalBorrowLimit,
 }) => {
+  const [selectedAsset, setSelectedAsset] = React.useState<Asset | undefined>(undefined);
   const styles = useStyles();
+
+  const rowOnClick: IBorrowMarketTableProps['rowOnClick'] | IBorrowingUiProps['rowOnClick'] = (
```

Comment:
Should all our market components mirror each other? The supply modals have their `rowOnClick` defined within the component.

Reply:
Yeah I saw that, but since the `rowOnClick` props and the `selectedAsset` state were being duplicated, I figured I could have all that here instead. We an do the same thing on the supply market table if you want

Reply:
Added to https://app.clickup.com/t/28vdcgd

Reply:
Ok sounds good 👍